WalletWasabi.Gui/Controls/NoparaPasswordBox.cs

```diff
@@ -381,8 +388,26 @@ private void PaintText()
 			{
 				GenerateNewRandomSequence();
 			}
+			var password = Sb.ToString();
+
+			var beforeTrim = password.Length;
+
+			password = password.Trim();
+
+			var whiteSpacesRemoved = beforeTrim != password.Length;
+			if (whiteSpacesRemoved)
+			{
+				Sb.Clear();
+				Sb.Append(password);
+			}
+
+			if (whiteSpacesRemoved)
+			{
+				_ = DisplayWarningAsync("Leading and trailing are removed!");
```

Comment:
"Leading and trailing white space was removed!" would make more sense imo

Reply:
How about "Leading and trailing white spaces were removed!"?

Reply:
Both work equally as well

Reply:
Yes it is a leftover mistake. For some reason if I paste the long pw to the pwbox with leading spaces instead of the warning message, empty pwbox appears. Strange.